viz/RobotModel.h

```diff
@@ -195,10 +195,17 @@ class RobotModel
     //this typedef define a pointer function to a load file function
     typedef osg::Node* (RobotModel::*LoadModelFunctionPtr)(QString path);
```

Comment:
Fixed. I also removed the "fix orientation of arm_shoulder_pan link" commit, it had nothing to do with this PR. Is it OK to merge now ?